mobile/src/navigations/main/main.tsx

```diff
@@ -1,30 +1,47 @@
-import {
-  type BottomTabNavigationOptions,
-  createBottomTabNavigator,
-} from '@react-navigation/bottom-tabs';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import React from 'react';
 
+import ChatIcon from '#assets/img/icons/chat.svg';
 import HomeIcon from '#assets/img/icons/home.svg';
+import { HeaderTitle } from '#libs/components/components';
 import { AppColor, MainScreenName } from '#libs/enums/enums';
 import { type TabNavigationParameterList } from '#libs/types/types';
-import { Home } from '#screens/main/home';
+import { Chat } from '#screens/chat/chat';
+import { Home } from '#screens/home/home';
 
 import { styles } from './styles';
 
 const BottomTab = createBottomTabNavigator<TabNavigationParameterList>();
 
-const tabNavigatorOptions: BottomTabNavigationOptions = {
-  tabBarActiveTintColor: AppColor.BLUE_200,
-  tabBarIcon: ({ color }) => <HomeIcon color={color} />,
-  tabBarInactiveTintColor: AppColor.GRAY_300,
+const tabNavigatorOptions = {
+  headerStyle: styles.headerStyle,
+  tabBarActiveTintColor: AppColor.BLUE_300,
+  tabBarInactiveTintColor: AppColor.GRAY_400,
   tabBarShowLabel: false,
   tabBarStyle: styles.tabBarStyle,
 };
 
 const Main: React.FC = () => {
   return (
     <BottomTab.Navigator screenOptions={tabNavigatorOptions}>
-      <BottomTab.Screen name={MainScreenName.HOME} component={Home} />
+      <BottomTab.Screen
+        name={MainScreenName.HOME}
+        component={Home}
+        options={{
+          headerTitle: () => <HeaderTitle title={MainScreenName.HOME} />,
+          tabBarIcon: ({ color }) => <HomeIcon color={color} />,
```

Comment:
```suggestion
          tabBarIcon: HomeIcon,
```
I think this should work

Reply:
Done